modules/describe-function-kn.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies
+
+// * /serverless/cli_reference/kn-func-ref.adoc
+
+[id="describe-function-kn_{context}"]
+= Describing a function
+
+The `kn func describe` command prints information for a deployed function, such as the function name, image, namespace, information about the Knative service, route information, and event subscriptions.
+
+.Procedure
+
+* Describe a function:
++
+[source,termnal]
+----
+$ kn func describe -f <format> -n <namespace> -p <path>
```

Comment:
Hmm.. I think here you can make the flags as optional parameters just the way you have on create and deploy commands.
```suggestion
$ kn func describe [-f <format> -n <namespace> -p <path>]
```